src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="mounth mounth--month-length--31">
+      <div class="mounth__day mounth__day--start-day--sun mounth__day--number-day--1">
+      </div>
+      <div class="mounth__day mounth__day--number-day--2"></div>
```

Comment:
I am not sure if each day should have its own modifier... I suppose modifiers should be on the whole calendar only (but not sure).

Reply:
it's necessary. because there is no content in the div, and the day numbers are assigned according to the class of the day